.github/workflows/release.yml

```diff
@@ -46,6 +46,17 @@ jobs:
         run: bash dev/ci-prepare.sh
       - name: "Make archives"
         run: python -u ./dev/releases/make_archives.py
+
+      - name: "Upload artifacts"
+        if: ${{ !startsWith(github.ref, 'refs/tags/v') }}
+        # Warning: the result is a single .zip file (so things are compressed twice)
+        # To keep things from exploding, we only upload a subset of all generated files
+        uses: actions/upload-artifact@v2
+        with:
+          path: |
+            tmp/gap-*.tar.gz*
+            tmp/*json*
```

Comment:
Note that one can also exclude patterns (see <https://github.com/actions/upload-artifact>), which can be useful to match `gap-X.Y.Z.tar.gz` but not `gap-X.Y.Z-core.tar.gz`

Reply:
I think adding `!tmp/gap-*-core.tar.gz*` should do the job to exclude `gap-X.Y.Z-core.tar.gz`

Reply:
```suggestion
            tmp/*json*
            !tmp/gap-*-core.tar.gz*
```